src/transformDataToAsset.js

```diff
@@ -1,34 +1,42 @@
-export default function(assetData) {
-    return {
-        Attributes: reduceAssetData(assetData)
-    };
-}
+import upperCamelCase from 'uppercamelcase';
 
-function reduceAssetData(obj) {
-    return Object.keys(obj).reduce((output, key) => {
+export default (assetData) => ({
+    Attributes: reduceAssetData(assetData)
+});
+
+const reduceAssetData = (obj) => Object.keys(obj)
+    .reduce((output, key) => {
         const attributeData = obj[key];
+        const assetAttributeKey = upperCamelCase(key);
         if (Array.isArray(attributeData)) {
-            output[key] = {
-                name: key,
-                value: attributeData.map(reduceRelationalAttributes)
+            return {
+                ...output,
```

Comment:
Wow...so I'm assuming "output" is an object here, and ...output in this context means
expand all its properties into this new literal obejct?